### PR TITLE
[ChangeItem] - calisthenic code part 1

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
simplify code

### Testing Instructions
code review

### CODE REVIEW

File: libraries/src/Schema/ChangeItem.php [hunk 1]
```diff
@@ -200,27 +200,20 @@ public function check()
...
-			else
+			if (count($rows) === $this->checkQueryExpected)
 			{
-				$this->checkStatus = -2;
+				$this->checkStatus = 1;
+				return $this->checkStatus;
 			}
+
+			$this->checkStatus = -2;
+
```
Comment (Quy) [inline libraries/src/Schema/ChangeItem.php]: Remove blank line

File: libraries/src/Schema/ChangeItem.php [hunk 2]
```diff
@@ -200,27 +200,20 @@ public function check()
 			}
 			catch (\RuntimeException $e)
 			{
-				$rows = false;
-
 				// Still render the error message from the Exception object
 				\JFactory::getApplication()->enqueueMessage($e->getMessage(), 'error');
+				$this->checkStatus = -2;
+				return $this->checkStatus;
```
Comment (Quy) [inline libraries/src/Schema/ChangeItem.php]: Add blank line before `return`

File: libraries/src/Schema/ChangeItem.php [hunk 3]
```diff
@@ -200,27 +200,20 @@ public function check()
...
-				else
-				{
-					$this->checkStatus = -2;
-				}
-			}
-			else
+			if (count($rows) === $this->checkQueryExpected)
 			{
-				$this->checkStatus = -2;
+				$this->checkStatus = 1;
```
Comment (Quy) [inline libraries/src/Schema/ChangeItem.php]: Add blank line before `return`

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on eb8ebe112b466367a484dd21cb0cd39ad8f26040<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19250">issues.joomla.org/tracker/joomla-cms/19250</a>.</sub>

File: libraries/src/Schema/ChangeItem.php (hunk 1 above)
Comment (alikon) [reply libraries/src/Schema/ChangeItem.php]: done

File: libraries/src/Schema/ChangeItem.php (hunk 2 above)
Comment (alikon) [reply libraries/src/Schema/ChangeItem.php]: done

File: libraries/src/Schema/ChangeItem.php (hunk 3 above)
Comment (alikon) [reply libraries/src/Schema/ChangeItem.php]: done

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9f19fcb0648c7f27cd62d8702907011ceb514822<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19250">issues.joomla.org/tracker/joomla-cms/19250</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19250">issues.joomla.org/tracker/joomla-cms/19250</a>.</sub>

**Pull request merged**